test/greet-test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const assert = require('assert');
+const greet = require('../lib/greet.js');
+
+describe('greet.js', function() {
+  describe('#sayHello', function() {
+    it('should return a string that greets the user by his or her name', function() {
+      assert.equal(greet.sayHello('Tommy'), 'Hello, Tommy!');
+    });
+    it('should return a string', function() {
+      assert.equal(typeof greet.sayHello('Jimmy'), 'string');
+    });
+    it('should process user input from the command line', function() {
+      assert.equal(greet.sayHello(process.argv[2]), `Hello, ${process.argv[2]}!`);
```

Comment:
Good attempt at testing for ``process.argv[2]``. However, this test at Line 15 is a bit of a false positive. The output from this will be ``Hello`` by itself without the ``process.argv[2]``. 

I know testing for it was a bit difficult to wrap your head around, but one way you can do this if you did something like:

```
process.argv[2] = 'judy';
```
And then Line 15 would be:

```
 assert.equal(greet.sayHello(process.argv[2]), `Hello, judy!`);
```